src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
 	<PropertyGroup>
-		<TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-android;net6.0-ios;net6.0-maccatalyst;net6.0-macos</TargetFrameworks>
```

Comment:
Would it be simpler to keep the support for WASM?

Reply:
Don't think , is it even working on WASM?

Reply:
It's Uno APIs so it should be working.

Reply:
netstandard2.0 was removed from Uno 5

Reply:
It's probably only .NET 7 then?

Reply:
True, I'll add net7.0 instead